scripts/plugins/Health.js

```diff
@@ -29,6 +29,7 @@ CivClicker.plugins.Health = (() => {
       this.tickSub = CivClicker.Events.subscribe('global.tick', () => {
         this.tick();
       });
+      this.temperatureSub = CivClicker.Events.subscribe('weather.temperature', this.temperature);
```

Comment:
Almost correct, but second argument should be an anonymous function, like this (using ES6 notation):

```
this.temperatureSub = CivClicker.Events.subscribe('weather.temperature', (temp) => {
  this.temperature = temp;
});
```

Reply:
Also don't forget to initialize `this.temperature` to something, like 0, in the plugin constructor.

Reply:
Cool. I thought the function was only for functions.